packages/charts/src/charts/choropleth/choropleth-chart.component.test.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import {
+  render as rtlRender,
+  fireEvent,
+  waitFor,
+} from '@testing-library/react';
+
+import ChoroplethChart from './choropleth-chart.component';
+
+const render = (overProps: any = {}) => {
+  const svgDimensions = {
+    width: 600,
+    height: 300,
+  };
+  const data = [{ 'geo.country': 'Poland', result: 1230 }];
+  const topology = {
```

Comment:
Please, move this map topology to separate fixture file to increase the readability.